### DISCUSSION TITLE
I rage-converted my RTX4090 into an eGPU | Tanel Poder Consulting

### DISCUSSION THREAD

**Discussion by giscus** [tanelpoder/blog-comments#41] [Announcements]
# I rage-converted my RTX4090 into an eGPU | Tanel Poder Consulting

One evening back in January I finally had enough of thermal issues within my homelab server. You know, every time the computer fans make more noise than I think they should, I can&rsquo;t help but investigate!
Also, the RTX4090 is so thick that it takes 3 PCIe slots worth of space on a typical motherboard - and you also need space for the airflow! I was using the same machine that achieved 11M IOPS &amp; 66 GB/s IO on a Single ThreadRipper Workstation back in 2020. - Linux, Oracle, SQL performance tuning and troubleshooting - consulting & training.

https://tanelpoder.com/posts/rtx4090-egpu/

**Comment by jk000079** [comment]
Nice. But why not pointing the exhaust up, to have an even better thermal flow efficiency?

**Reply by tanelpoder** [reply]
This specific card pulls air in from one side and pushes all of it out to the other side (plus some air goes out to the back where the HDMI/Displayport sockets are). So if I changed the fan to blow hot air upwards, the intake side would have limited space. So currently it's perfect given my constraints - it pulls colder air in from the left (on the video) and blows it out to the right... hot air moves upwards anyway and there's no intake on the hot side. At some point I'll blog about the bigger picture too, how I get warm air out of my "data centre".
